lib/cosmos/tools/test_runner/test.rb

```diff
@@ -103,10 +103,6 @@ def self.test_cases
           test_cases << method_name.to_s
         end
       end
-
-      # Sort by name
-      test_cases.sort!
```

Comment:
I think this should remain.  The test cases for a Test by default run in alphabetical order and should be returned sorted.

Reply:
I believe this new method inserts the test cases as they are defined in the file which is what most people intuitively expect. I'm not sure if this is guaranteed but it appears to be the case as the interpreter populates instance_methods as they are encountered.

Reply:
Removing this sort will change the order that test cases run in existing test procedures (if they were not defined in alphabetical order).  I see that as bad.

Reply:
I also don't like relying on an interpreter implementation detail (that instance_methods happens to keep them in defined order).